Layout.py

```diff
@@ -5,15 +5,20 @@
 
 
 PLUGIN_NAME = 'Layout'
-PLUGIN_PATH = os.path.join(sublime.packages_path(), PLUGIN_NAME)
-LAYOUTS_PATH = os.path.join(PLUGIN_PATH, 'layouts')
-
 X_MIN, Y_MIN, X_MAX, Y_MAX = (0, 1, 2, 3)
 
 
+def plugin_path():
+    return os.path.join(sublime.packages_path(), PLUGIN_NAME)
+
+
+def layouts_path():
+    return os.path.join(plugin_path(), 'layouts')
+
+
```

Comment:
I moved is functions in their own definitions, instead of calling them from global vars (I often get errors when loading files if they are in that position)